<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListedPCGame</class>
 <widget class="QWidget" name="ListedPCGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>25</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>25</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	border-color : rgb(0, 0, 0);
	border-width : 1px;
	border-style: solid;
	background-color: rgba(86, 86, 86, 100);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelLogo">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/Images/windowslogo</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelGameName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Game Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPlatform">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Platform</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAdd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox:!checked
{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(125, 125, 125, 255), stop:1 rgba(0, 0, 0, 255));
}
QCheckBox:checked
{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0502513 rgba(0, 80, 120, 255), stop:0.58794 rgba(0, 0, 127, 255));
}

QCheckBox::indicator
{
	width: 10px;
	height: 10px;
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.1 rgba(4, 4, 4, 255), stop:0.2 rgba(32, 32, 32, 215), stop:0.3 rgba(0, 0, 0, 201), stop:0.4 rgba(0, 0, 0, 229), stop:0.5 rgba(21, 21, 21, 255), stop:0.52 rgba(0, 0, 0, 255), stop:0.6 rgba(0, 0, 0, 84), stop:1 rgba(18, 18, 18, 214));
}
QCheckBox::indicator:checked
{
	position: absolute;
	left: 45px;
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(72, 146, 255, 255), stop:0.1 rgba(87, 136, 200, 255), stop:0.2 rgba(87, 111, 218, 215), stop:0.3 rgba(51, 100, 186, 201), stop:0.417085 rgba(56, 50, 214, 229), stop:0.462312 rgba(52, 66, 235, 255), stop:0.517588 rgba(36, 57, 183, 255), stop:0.572864 rgba(44, 37, 228, 255), stop:0.643216 rgba(29, 43, 220, 84), stop:1 rgba(36, 21, 232, 214));
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
